consensus/bor/bor.go

```diff
@@ -436,9 +432,10 @@ func (c *Bor) verifyCascadingFields(chain consensus.ChainReader, header *types.H
 		return err
 	}
 
-	isSprintEnd := (number+1)%c.config.Sprint == 0
 	// verify the validator list in the last sprint block
-	if isSprintEnd {
+	if isSprintStart(number, c.config.Sprint) {
+		parentHeader := chain.GetHeaderByNumber(number - 1)
```

Comment:
There is already `parent` available in this function. We don't need `parentHeader`